### DISCUSSION TITLE
Is it possible to control the tray icon size?

### DISCUSSION THREAD

**Discussion by ddogfoodd** [Alexays/Waybar#3547] [Q&A] [answered]
Is there a config option to set the size of icons that are in the tray?
I looked through the default config and tried some things, but didn't find anything.

**Comment by dglava** [comment, marked as answer]
Yes, look at the Wiki entry for the Tray module.
https://github.com/Alexays/Waybar/wiki/Module:-Tray

**Reply by ddogfoodd** [reply]
Thanks somehow overlooked that.
